Multisig/UI/App/RemoteNotificationHandler.swift

```diff
@@ -168,8 +168,8 @@ class RemoteNotificationHandler {
             let appConfig = App.configuration.app
             var timestamp: String?
             if let _ = try? App.shared.keychainService.data(forKey: KeychainKey.ownerPrivateKey.rawValue) {
-                // add timestamp if there is a signing key
-                timestamp = String(Int(Date().timeIntervalSince1970 * 1_000))
+                // add timestamp in seconds if there is a signing key
```

Comment:
now this comment is not needed, the code says it

Reply:
ok